english/story-continues

```diff
@@ -0,0 +1 @@
+The story continues.  You wake up and it was all a dream.  Then, in the corner of your eye you see [a] (http://ancientexplorers.com/wp-content/uploads/sites/6/2015/12/sparkling-blue-universe_131922.jpg)  
```

Comment:
It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.